platforms/nuget/TUTORIAL.md

```diff
@@ -0,0 +1,39 @@
+# Using the NuGet Package (OpenCVNuget) in your native C++ project
+This tutorial takes you through downloading the .nupkg file and setting it up in your C++ project, so you can begin writing C++ applications with OpenCV quickly.
+
+## Downloading the latest .nupkg (NuGet Package) file
+
+- You can download the package from
+    - The releases section on Github
+    (https://github.com/opencv/opencv/releases)
+    - The releases section on OpenCV website
+    (https://opencv.org/releases/)
+- Ensure that you have the correct package based on your Visual Studio or Microsoft Visual C++ Version. Currently supported versions are:
```

Comment:
For the purposes of the GSOC project I would remove the possible future location of the nuget and add a link to the current location of the action.

Reply:
This has been changed to the project repository's release page (https://github.com/AdityaMulgundkar/opencv-nuget/releases), since I will have to maintain / keep the GSoC project repository live as part of the project report.